input/pagecontent/sushi.md

```diff
@@ -0,0 +1,130 @@
+
+> **NOTE**: HL7® and FHIR® are registered trademarks owned by Health Level Seven International, and are registered with the United States Patent and Trademark Office.
+
+### Purpose
+
+This reference manual is a comprehensive guide to the command line interface, auxiliary files, and configurations needed to create an HL7 FHIR IG using SUSHI.
+
+### Intended Audience
+
+The reference manual is targeted to people doing IG development using FSH. Familiarity with FHIR is helpful as the manual references various FHIR concepts (profiles, extensions, value sets, etc.)
+
+### Prerequisite
+
+This guide assumes you have:
+
+* Created FSH files representing your profiles and other IG artifacts (see [FHIR Shorthand Reference Manual](shorthand.md) for details).
+* Reviewed the [FHIR Shorthand Tutorial](tutorial.md).
+
+### Document Conventions
+
+| Style | Explanation | Example |
+|:----------|:------|:---------|
+| `Code` | Code fragments, such as commands and Shorthand statements  | `* status = #open` |
+| _italics_ | File names | _example-1.fsh_ |
+| {curly braces} | An item to be substituted | `* status = {code}` |
+| **bold** | General emphasis |  Do **not** fold, spindle or mutilate. |
+
+We use `$` to represent the command prompt, although that may differ on your operating system.
+
+## Installation
+Install SUSHI in two easy steps:
+
+1) Install Node.js
+    
+    a) Download from https://nodejs.org 
+    
+    b) Run the installer
+
+    c) Check the installation by opening a command window and typing the following two commands. Each command should return a version number. 
+
+    `$ node --version`
+
+    `$ npm --version`
+
+1) Install SUSHI
+
+    a) Issue this command at the prompt:
+
+    `$ npm install -g fsh-sushi`
+
+    b) Check the installation by typing the following command:
+
+    `$ sushi -h`
+
+If the command outputs instructions on using SUSHI command line interface (CLI), you're ready to run SUSHI.
+
+## SUSHI Version
+
+SUSHI follows the [semantic versioning](https://semver.org) convention (MAJOR.MINOR.PATCH):
+
+* MAJOR: A major release has significant new functionality and potentially, grammar changes or other non-backward-compatible changes.
+* MINOR: Contains new or modified features, while maintaining backwards compatibility within the major version.
+* PATCH: Contains minor updates and bug fixes, while maintaining backwards compatibility within the major version.
+
+_Is this correct?_
+
+> The config file must declare the minimum version of SUSHI compiler it requires. For example, if the config declares SUSHI 1.2, it requires major version 1, any minor version >= 2.
+
+>**NOTE:** There is no guarantee that a FSH file declaring major version X will compile correctly on major version Y, if X /= Y.
+
+_end_
```

Comment:
I think this can be correct if we want it to be, but isn't yet something we check for.